docs/source/developers/python.rst

```diff
@@ -378,20 +378,8 @@ Docker examples
 
 If you are having difficulty building the Python library from source, take a
 look at the ``python/examples/minimal_build`` directory which illustrates a
-complete build and test from source both with the conda and pip/virtualenv
-build methods.
-
-Building with CUDA support
-~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-The :mod:`pyarrow.cuda` module offers support for using Arrow platform
-components with Nvidia's CUDA-enabled GPU devices. To build with this support,
-pass ``-DARROW_CUDA=ON`` when building the C++ libraries, and set the following
-environment variable when building pyarrow:
```

Comment:
This section is also included elsewhere?

Reply:
It's simply covered by the bulleted list of optional components above. There is no need for a CUDA-specific paragraph IMHO.